src/shared/schemas/wallet.ts

```diff
@@ -7,4 +26,10 @@ export enum WalletType {
   smartAwsKms = "smart:aws-kms",
   smartGcpKms = "smart:gcp-kms",
   smartLocal = "smart:local",
+
+  //breaking
```

Comment:
Not sure what you mean by this comment. This shouldn't be a breaking change?

Reply:
yeah should be clearer there, breaking as in: only supports the newer WalletCredential based creation